common/starbase_modules/HFE_TSP_drone_star_base_module.txt

```diff
@@ -0,0 +1,130 @@
+#
+
+hfe_starbase_module_trading_hub = {
+	icon = "GFX_starbase_offworld_trading_company"
+	section = "REFINERY_STARBASE_SECTION"
+	construction_days = 1
+	initial = yes
+
+	potential = {
+		solar_system = {
+			has_star_flag = hfe_drone_economic_zone
+		}
+	}
+
+	possible = {
+		owner = {
+			OR = {
+				is_country_type = hfe_drone
+				is_country_type = drone
+			}
+		}
+		owner = {
```

Comment:
It accepts only specific country type, no need in this trigger.

Reply:
Is it causing error logs?

Reply:
Oh please.